FilteringApp.sln

```diff
@@ -11,9 +11,9 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FilteringApp.Core", "Filter
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "GCD", "GCD", "{A1FD278A-5635-44D0-9EFC-286EA44DA7F6}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GrandCommonDivisor.ConsoleUI", "GrandCommonDivisor\GrandCommonDivisor.ConsoleUI.csproj", "{897C70BC-A580-40C9-A8D6-563706AF89EB}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GrandCommonDivisor.ConsoleUI", "GCD\GrandCommonDivisor\GrandCommonDivisor.ConsoleUI.csproj", "{897C70BC-A580-40C9-A8D6-563706AF89EB}"
```

Comment:
![image](https://user-images.githubusercontent.com/23036313/145696907-472c99b8-d763-40a8-aa54-6359e4e902d7.png)
It seems that there are still leftovers in the root folder

Reply:
Nevermind, it seems that it was some issue on my local machine. I'm closing this one